tests/functional/modules/test_zos_job_output_func.py

```diff
@@ -82,5 +82,19 @@ def test_zos_job_output_job_exists(ansible_zos_module):
     hosts.all.file(path=TEMP_PATH, state="absent")
     results = hosts.all.zos_job_output(job_name="SAMPLE")
     for result in results.contacted.values():
+        print(result)
+        assert result.get("changed") is False
+        assert result.get("jobs") is not None
+
+
+def test_zos_job_output_nonexistant(ansible_zos_module):
```

Comment:
How are you handling the plural version of your error message for query , "No jobs found"

Reply:
I'm not sure how to replicate that... it should only trigger if NO jobs are returned on the global query... while it's a possibility, I haven't seen it happen... any time I've run it there has been SOMETHING in the JES Queue.